src/plugins.js

```diff
@@ -104,12 +104,12 @@ function getServiceWorkerPlugin() {
 }
 
 export default function getPlugins() {
-	const { platform_agent, isProd } = config;
```

Comment:
This seems to be the culprit for the failed unit test.

We should either mock or add a config.agent to mwp's `package.json`